pytest_tests/testsuites/object/test_object_api.py

```diff
@@ -337,6 +341,69 @@ def test_object_search_should_return_tombstone_items(
                     object_type == "TOMBSTONE"
                 ), f"Object wasn't deleted properly. Found object {tombstone_oid} with type {object_type}"
 
+    @allure.title("Validate objects search by common prefix")
+    def test_search_object_api_common_prefix(self, default_wallet: str, simple_object_size: int):
+        FILEPATH_ATTR_NAME = "FilePath"
```

Comment:
Maybe it would be better to move it to common?

Reply:
I don't think so - it is not some 'special' attribute, you can use abrakadabra as well here